airbyte-integrations/connectors/source-bing-ads/source_bing_ads/source.py

```diff
@@ -313,6 +313,187 @@ def stream_slices(
         yield from []
 
 
+class BudgetSummaryReport(ReportsMixin, BingAdsStream):
+    primary_key = None
```

Comment:
I think it better to move `primary_key = None` to the parent class` BingAdsStream` and add `primary_key = "Id"` to the other three streams implemented before.